courses/management/commands/manage_certificates.py

```diff
@@ -0,0 +1,189 @@
+"""
+Management command to revoke and un revoke a certificate for a course run or program for the given user.
```

Comment:
Could you add here also the certificate created and update of a grade for a user to the description?

Reply:
Updated the documentation and added usages too.